README.md

```diff
@@ -101,9 +101,15 @@ To enable GPU acceleration for `hailo` conversion, install the [Nvidia Container
 
 ## Running ModelConverter
 
-Configuration for the conversion predominantly relies on a `yaml` config file. For reference, see [defaults.yaml](shared_with_container/configs/defaults.yaml) and other examples located in the [shared_with_container/configs](shared_with_container/configs) directory.
+There are two main ways to execute configure the conversion process:
 
-However, you have the flexibility to modify specific settings without altering the config file itself. This is done using command line arguments. You provide the arguments in the form of `key value` pairs. For better understanding, see [Examples](#examples).
+1. **YAML Config File (Primary Method)**:
+   The primary way to configure the conversion is through a YAML configuration file. For reference, you can check [defaults.yaml](shared_with_container/configs/defaults.yaml) and other examples located in the [shared_with_container/configs](shared_with_container/configs) directory.
+1. **NN Archive**:
```

Comment:
IMO conversion through YAML should be the preferred method since this is it's intention. Conversion through intermediate NNArchive is more like an added bonus since NNarchive itself holds only information about the model it is tied to and not about how to convert it to the next model - this is why we need to have assumptions built in when using intermediate NNArchive as input to the conversion (e.g. assumption that exported model should be BGR where as with config.yaml you can change this to RGB if you want)